frontend/src/component/Call/Call.jsx

```diff
@@ -0,0 +1,227 @@
+
+  let unansweredCallTimeoutId = -1
+
+  useEffect(() => {
+    props.registerLiveMessageHandlerList([
+      { entityType: TLM_ET.USER_CALL, coreEntityType: TLM_CET.MODIFIED, handler: handleUserCallModified },
+      { entityType: TLM_ET.USER_CALL, coreEntityType: TLM_CET.CREATED, handler: handleUserCallCreated }
+    ])
+    setIsMainTab(props.liveMessageManager.eventSource !== null)
+  }, [])
+
+  useEffect(() => {
+    setIsMainTab(props.liveMessageManager.eventSource !== null)
+  }, [props.liveMessageManager.eventSource])
+
+  const handleUserCallCreated = async (tlm) => {
+    if (tlm.fields.user_call.callee.user_id === props.user.userId) {
+      if (window.Notification) {
+        const notificationString = tlm.fields.user_call.caller.public_name + props.t(' is calling you on Tracim')
+        const notificationOptions = { tag: 'call', renotify: true, requireInteraction: true }
+
+        try {
+          if (Notification.permission === 'granted') {
+            new Notification(notificationString, notificationOptions) // eslint-disable-line no-new
+          } else if (Notification.permission !== 'denied') {
+            const permission = await Notification.requestPermission()
+            if (permission === 'granted') {
+              new Notification(notificationString, notificationOptions) // eslint-disable-line no-new
+            }
+          }
+        } catch (e) {
+          console.error('Could not show notification', e)
+        }
+      }
+
+      setUserCall(tlm.fields.user_call)
+      props.dispatch(setHeadTitle(props.system.headTitle, '🔔'))
+    } else if (tlm.fields.user_call.caller.user_id === props.user.userId) {
+      setUserCall(tlm.fields.user_call)
+    }
+  }
+
+  const handleUserCallModified = (tlm) => {
+    if (tlm.fields.user_call.callee.user_id === props.user.userId) {
+      setUserCall(defaultUserCall)
+
+      const callStateForPauseAudio = [
+        USER_CALL_STATE.ACCEPTED,
+        USER_CALL_STATE.CANCELLED,
+        USER_CALL_STATE.REJECTED,
+        USER_CALL_STATE.UNANSWERED,
+        USER_CALL_STATE.DECLINED
+      ]
+      if (callStateForPauseAudio.some(state => state === tlm.fields.user_call.state)) {
+        audioCall.pause()
+        props.dispatch(setHeadTitle(props.system.headTitle))
+      }
+    }
+
+    if (tlm.fields.user_call.caller.user_id === props.user.userId) {
+      clearTimeout(unansweredCallTimeoutId)
+
+      setUserCall(tlm.fields.user_call)
+      unansweredCallTimeoutId = -1
+
+      if (tlm.fields.user_call.state === USER_CALL_STATE.ACCEPTED) {
+        if (!isMainTab) return
+        window.open(tlm.fields.user_call.url)
+      }
+    }
+  }
+
+  const handleClickOpenCallWindowCallee = () => {
+    props.dispatch(putSetIncomingUserCallState(props.user.userId, userCall.call_id, USER_CALL_STATE.ACCEPTED))
+    props.dispatch(setHeadTitle(props.system.headTitle))
+    audioCall.pause()
+  }
+
+  const handleClickRejectCall = () => {
```

Comment:
isn't it the same as handleUserCallModified, shouldn't only one handler be use for :
 - props.dispatch(setHeadTitle(props.system.headTitle))
 - audioCall.pause()

Reply:
Indeed, will test it and will fix